src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -86,7 +86,15 @@
     static {
         Properties prop = new Properties();
         try {
-            prop.load(new FileInputStream("src/test/resources/test.properties"));
+            String propertiesFilename;
+            if (isTravis()) {
+                propertiesFilename = "test.travis.properties";
+            } else if (isAppveyor()) {
+                propertiesFilename = "test.appveyor.properties";
+            } else {
+                propertiesFilename = "test.properties";
+            }
+            prop.load(new FileInputStream("src/test/resources/" + propertiesFilename));
```

Comment:
I don't like the idea of putting build-specific configuration values inside Java code. We should allow easy tweaking via build config files without having to modify code.